tests/integration/test_httpx.py

```diff
@@ -314,3 +319,19 @@ def test_redirect_wo_allow_redirects(do_request, mockbin, yml):
         assert response.status_code == 308
 
         assert cassette.play_count == 1
+
+@pytest.mark.online
+@pytest.mark.parametrize("url", ["https://github.com/kevin1024/vcrpy/issues/" + str(i) for i in range(3, 6)])
```

Comment:
It would be great if we could use the httpbin fixture so that we do not need real internet for this test. E.g. we could use `url = f'{httpbin.url}/bytes/<some number>'` and then check if what we read matches that length. The name of the fixture is just `httpbin`. `@pytest.mark.online` could then be dropped. What do you think?

Reply:
Great idea! I don't have a ton of httpbin experience, but that works. I'll submit an update.